Platforms/Docs/Common/Features/feature_memoryprotection.md

```diff
@@ -1,12 +1,24 @@
 # Memory Protection
 
-For in-depth information on Memory Protection in Project Mu, see
-[feature_memory_protection.md](../../../../MU_BASECORE/Docs/feature_memory_protection.md)
+For in-depth technical details on Memory Protection in Project Mu, see
+[feature_memory_protection.md](https://github.com/microsoft/mu_basecore/blob/HEAD/Docs/feature_memory_protection.md)
 
-Memory protection is ON by default on Q35 and SBSA. To disable memory protection
-add `BLD_*_MEMORY_PROTECTION=FALSE` to your `stuart_build` command. Example:
+Memory protections are important because Unified Extensible Firmware Interface (UEFI) standard accounts for the
+firmware design implemented in 80 to 90 percent of the of PCs and servers sold worldwide. Developed and supported by
+more than 250 industry-leading companies, UEFI firmware is responsible for booting and securing billions of devices
+spanning device classes from embedded applications to multi-role server systems.
+
+While considerable attention has been devoted to hardware trust anchors and operating system security, attackers have
+discovered that UEFI firmware is lacking basic memory protections that have been present in other system software for
+over a decade. Coupled with the inconsistency of security capabilities inherit to vendor firmware implementations,
+UEFI firmware has become an increasingly attractive system attack vector.
+
+`QemuQ35Pkg` allows experimentation with the memory protections being offered in physical platforms.

```

Comment:
nit: so does QemuSbsaPkg

Reply:
Thanks, I'll update that. This entire documentation format didn't really scale super well to QemuSbsaPkg as the file is here and there's a number of features over there.

I don't have time at the moment to clean up covering both, but I'll file an issue for it.

Reply:
Filed in #675 and mentioned QemuSbsaPkg in the latest push.